cve_bin_tool/version_scanner.py

```diff
@@ -138,13 +129,32 @@ def scan_file(self, filename):
             )
         else:
             # Otherwise, use python implementation
-            lines = Strings(filename).parse()
-        #  If python package then strip the lines to avoid detecting other product strings
-        if "PKG-INFO: " in output or "METADATA: " in output:
-            lines = lines[1:3]
-            lines[0] = (
-                "--generated pattern for cve-bin-tool " + lines[0] + " " + lines[1]
```

Comment:
Is this going to break the package scanner?  It looks to me like it might have been an overzealous merge fix.